docs/contributing/testing.md

```diff
@@ -0,0 +1,112 @@
+# Test your GOV.UK Frontend contribution
+
+Before you submit your contribution, you should:
+
+- check your changes look right and behave correctly
+- test in supported browsers and assistive technology
+- run our automated tests, and update tests or add new tests if our tests fail
+
+[Contact us](https://design-system.service.gov.uk/#support) for help if either:
+
+- you can’t do some of the checks or tests
+- your contribution might affect other parts of GOV.UK Frontend
+
+## 1. Lint your Sass and JavaScript
+
+Do the following to lint your Sass code.
+
+1. Install a linter in your code editor, for example [linter-sass-lint for Atom](https://atom.io/packages/linter-sass-lint).
```

Comment:
This is an optional convenience that makes sense if you're working on the project for an extended period of time, most people can get away with running `npm test` before they push their changes to the server. Should we consider making this section optional?